mobile/src/main/res/layout/widgetlist_sectionswitchitem_compact.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    style="@style/WidgetListItemContainerCompact">
+
+    <include layout="@layout/widgetlist_icontext_compact" />
+
+    <com.google.android.material.button.MaterialButtonToggleGroup
+        android:id="@+id/switch_group"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="4dp"
+        app:singleSelection="true" />
```

Comment:
When the widgetLabel is hidden, these buttons are left aligned.

Reply:
Note to self: Place a `Space` in front of the buttons with `layout_weight` > 0